_pytest/capture.py

```diff
@@ -308,6 +308,14 @@ class FDCapture:
     """ Capture IO to/from a given os-level filedescriptor. """
 
     def __init__(self, targetfd, tmpfile=None):
+        # ensure readline is imported so that it attaches to the correct
+        # stdio handles on Windows
+        if targetfd in (0, 1, 2):
+            try:
+                import readline
```

Comment:
Okay, thanks.  I added the comment specifically to point out that the unused import has a purpose.